app/views/patients/_patient_information.html.erb

```diff
@@ -74,7 +74,7 @@
           </div>
 
           <%= f.select :insurance,
-                       options_for_select(insurance_options,
+                       options_for_select(insurance_options.push(patient.insurance).uniq,
                                           patient.insurance),
```

Comment:
My thought here was that we don't want to accidentally overwrite an insurance option if it disappears from the dropdown and the form saves, so if a patient has an insurance option that isn't present anymore, it'll just append it to the end of the list. Not very clear from the code here, so could probably use an explanation, though!